static/app/views/issueDetails/streamline/header/header.tsx

```diff
@@ -163,13 +147,34 @@ export default function StreamlinedGroupHeader({
               ))}
           </StatTitle>
           <Flex gap={space(1)} align="center" justify="flex-start">
+            {issueTypeConfig.logLevel.enabled && (
+              <ErrorLevel level={group.level} size={10} />
+            )}
+            <SecondaryTitle
+              title={secondaryTitle}
+              isHoverable
+              showOnlyOnOverflow
+              delay={1000}
+              isDefault={!secondaryTitle}
+            >
+              {secondaryTitle ?? t('No error message')}
```

Comment:
Hmm this isn't working for me when there is no message (might be because it's an empty string?)

is it possible to use the `<EventMessage />` component here?

![CleanShot 2025-03-24 at 14 32 50](https://github.com/user-attachments/assets/3a7ba6b3-93c4-401b-a7de-61075c31f185)